### PR TITLE
Action for building windows vs2017 base images

### PR DESCRIPTION
**Pull request by assignUser**
This version is not caching between runs but enables us to rebuilt the base images, e.g. for PRs.

### CODE REVIEW

File: .github/workflows/cache_vcpkg.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
           else
             vcpkg_version="${{ github.event.inputs.vcpkg-version }}"
           fi
-
+
```
Comment (assignUser) [inline .github/workflows/cache_vcpkg.yml:76]:
```suggestion

```

File: .github/workflows/windows_docker.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+jobs:
+  build:
+    name: "Build Windows images"
+    runs-on: windows-2019
+    permissions:
+      contents: read
+      packages: write
+    env:
+      # Turned on by default in .env but Windows does not support Buildkite
```
Comment (kszucs) [inline .github/workflows/windows_docker.yml:25]:
```suggestion
      # Turned on by default in .env but Windows does not support buildkit

```

**Comment by kszucs** [comment]
@assignUser have you tried to trigger this workflow? I'm curious of how much time it takes to build all 4 images.

**Comment by assignUser** [comment]
@kszucs Yes, it takes about 2.5h: https://github.com/assignUser/test-repo-a/runs/6115203943?check_suite_focus=true

**Review by kszucs — APPROVED** [review]
Great, thanks @assignUser!

**Pull request merged**